src/styles/main.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import "./utilit/veriables";
+@import "./utilit/mixin";
+@import "./blocks/header";
+@import "./blocks/card";
+@import "./blocks/stars";
+@import "./blocks/catalog";
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
```

Comment:
```suggestion

```
Consider moving general page styles to a separate file to keep your main file clean and clear